Woof/Woof/Helper/AppAlert/AppAlert.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Represents alerts which uses in application.
+enum AppAlert: String, CustomStringConvertible, CaseIterable {
```

Comment:
Why do we need an _enum_? It's important to be sure that we choose among restrict amount of cases? I don't really understand a purpose to adopt `CustomStringConvertible`. It seems as an overengineering.

Reply:
Agree with @kirpichonok that we can use struct. We don't have to call the rawValue to get access to the string or make it static let in enum as an option. WDY?

Reply:
We can use _static_ properties of an _Enum_ or a _Structure_. These constants are not strictly restricted set.

Reply:
Fixed with 03725e9